system/Database/SQLite3/Connection.php

```diff
@@ -56,6 +57,15 @@ class Connection extends BaseConnection
      */
     protected $busyTimeout;
 
+    /**
+     * The setting of the "synchronous" flag
+     *
+     * @var int|null flag
```

Comment:
Based on the review of @maniaba, this accepts only `0` to `3` so might be worthwhile to help phpstan here.
```suggestion
     * @var int<0, 3>|null flag
```